### ISSUE TITLE
Phylip format file

### ISSUE THREAD

**Issue by heatherturtle** [sansubs/vcf2pop#2]
Hello.  It says that vcf2pop exports a Phylip file although, I do not see any outputted file.  How do I get the diversity matrix off of the html website so it can be used with phangorn?

**Comment by sansubs** [comment]
Yes.  That's correct.  Just copy and past it into a text editor.  This could be used as the input phylip file for any software that creates a tree using this.  Sorry for the belated reply.
